vcpkg/overlay/qgis/portfile.cmake

```
@@ -19,7 +19,8 @@ vcpkg_from_github(
         bigobj.patch
         android-resources.patch
         poly2tri.patch
-        inverted.patch
+        pgprovider.patch
+        qgsvariantutils.patch
```

```suggestion
        qgsvariantutils.patch # https://github.com/qgis/QGIS/pull/50662
```

This will make it easier to decide to remove this patch again in the future

---

@m-kuhn good idea 💡